syntax/pandoc.vim

```diff
@@ -223,6 +224,14 @@ endif
 
 " Syntax Rules: {{{1
 
+" corrects distracting conceal (ligature) background colors, blue makes 
+" it obvious that the character is a ligature instead of an actual Unicode
+" character since both can exist in the same file
+hi Conceal ctermbg=none ctermfg=Blue
```

Comment:
Could this be linked to an existing highlight group of some sort rather than hard coding a color? That way it will play nice with themes. `Special`, `Macro`, or `Character` might work.

Reply:
Actually I think this might be better left to color schemes entirely. I get that the vim defaults are pretty bad, but for any color scheme that _does_ try to set the colors for `Conceal` having a plugin hard code an override for that is kind of rough luck. I think I'd rather see this bit contributed to whatever colorscheme you use and/or added to users' vim-rc file.